site/build-and-publish.sh

```diff
@@ -13,6 +13,10 @@ if [ -n "${TRAVIS_TAG}" ]; then
   OUTPUT="weave/${TRAVIS_TAG}"
 fi
 
+if [ "${BRANCH}" = "latest_release_doc_updates" ]; then
```

Comment:
I get this shouldn't be master, because it would pull in code changes before release, but I worry this branch will be neglected/forgotten.